hx_lti_assignment/templates/hx_lti_assignment/create_new_assignment2.html

```diff
@@ -21,76 +21,72 @@
 			<p>You'll need some source materials to create an annotation assignment!</p>
 		{% else %}
 			<div class='save' id='reorder-list-button'>Reorder list</div>
-			{% for source in targets_form %}
-				{% for choice in form.fields.assignment_objects.queryset %}
-					{% if source.target_object.value == choice.id %}
-						<div class='source-item' id='item-{{forloop.parentloop.counter0}}' data-order='{{source.order.value}}' data-id='{{source.target_object.value}}' data-aTarget="{{source.id.value}}" data-media="{{source.target_type.value}}">
-							<div class='first-row'>
-								<i class='fa fa-caret-right fa-lg' style='color:rgb(55, 124, 182);'></i> <span class='source-title'>{{choice.target_title}}</span>
-								<div class='order-div'><input type="number" class='ordernum' style='display:none;' min="1" value="{{source.order.value}}"><a href='#' style='display: none' class='order-change-button'>Move</a></div>
+			{% for source in form.instance.assignmenttargets_set.all %}
```

Comment:
Change ```form.instance.assignmenttargets_set.all``` to ```targets_form```.

Every instance of "source.[something]" i.e. ```source.order``` or ```source.target_object.target_title``` should be changed to "source.instance.[something]" i.e. ```source.instance.order``` or ```source.instance.target_object.target_title```.

I think that should do it. Can you test that. Add it to this PR and I can merge this to admin_hub and we can test for a day or two and then merge to master?